pwa/src/templates/templateParts/submitComponent/SubmitComponentTemplate.module.css

```diff
@@ -0,0 +1,32 @@
+.container {
+  display: flex;
+  background-color: var(--web-app-submit-component-background-color);
+  padding-inline-start: var(--web-app-size-md);
+  padding-inline-end: var(--web-app-size-md);
+  padding-block-start: var(--web-app-size-md);
+  padding-block-end: var(--web-app-size-md);
+}
+
+.container > *:not(:last-child) {
+  margin-block-end: var(--web-app-size-md);
+}
+
+.header {
+  flex: 2;
+}
+
+.title {
+  font-weight: bold;
+  font-size: var(--web-app-font-size-xl);
+  margin-block-end: var(--web-app-size-xs);
+}
+
+.buttonContainer {
+  flex: 1;
+  align-self: center;
+  text-align: -webkit-right;
```

Comment:
We can do this without text-align or align-self. It should be positioning its children (the button), like so:

```
display: flex;
justify-content: flex-end;
align-items: center;
```